scripts/install_odf.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+ibmcloud config --check-version=false
+ibmcloud login -apikey ${IC_API_KEY}
+ibmcloud ks cluster config -c ${CLUSTER} --admin
+
+# Addons variables assigments
+
+
+# Retrieve the openshift cluster version
+ROKS_VERSION=`oc get clusterversion -o jsonpath='{.items[].status.history[].version}{"\n"}'`
```

Comment:
Use `kubectl` instead of `oc`.  We can not require a user to download the `oc` command since they might not have an OpenShift account to download it from.